acceptance_tests/features/steps/rh_ui_cookies.py

```diff
@@ -0,0 +1,110 @@
+        "ons-js-accept-cookies" if action == "accepts" else "ons-js-reject-cookies"
+    )
+
+    xpath_string = f"//button[contains(@class,{button_class})]"
+    context.browser.find_by_xpath(xpath_string).click()
+
+
+@step("the 'View cookies' hyperlink points to {expected_path}")
+def click_view_cookies_link_on_banner(context, expected_path):
+    xpath_string = '//a[@class="ons-cookies-banner__link" and text()="View cookies"]'
+    link = context.browser.find_by_xpath(xpath_string).first
+    test_helper.assertEqual(link["href"], f"{Config.RH_UI_URL}{expected_path}")
+
+
+@step("the 'cookies' hyperlink on the cookies banner points to {expected_path}")
+def click_cookies_hyperlinked_text_on_banner(context, expected_path):
+    xpath_string = '//div[@class="ons-cookies-banner__statement"]/a[text()=" cookies."]'
+    link = context.browser.find_by_xpath(xpath_string)
+    test_helper.assertEqual(link["href"], f"{Config.RH_UI_URL}{expected_path}")
+
+
+@step('the "change cookie preferences" hyperlink text points to {expected_path}')
+def click_change_cookie_preferences_link_on_banner(context, expected_path):
+    xpath_string = '//span[@class="ons-cookies-banner__preferences-text"]/a[text()=" change your cookie preferences"]'
+    link = context.browser.find_by_xpath(xpath_string)
+    test_helper.assertEqual(link["href"], f"{Config.RH_UI_URL}{expected_path}")
+
+
+# Cookies page radio interactions
+
+
+@step("the user sets the selection under {para_title} to {cookie_selection}")
+def change_cookies_selection(context, para_title, cookie_selection):
+    match para_title:
+        case "Cookies that measure website use":
+            name_attribute_value = "cookies-usage"
+        case "Cookies that help with our communications":
+            name_attribute_value = "cookies-campaigns"
+        case "Cookies that remember your settings":
+            name_attribute_value = "cookies-settings"
+        case _:
+            raise Exception("Expected header not found on the cookies page")
+
+    xpath_string = f'//input[@name="{name_attribute_value}" and @value="{cookie_selection.lower()}"]'
+    context.browser.find_by_xpath(xpath_string).first.click()
+
+    submit_button_xpath_string = '//button[@type="submit"]'
+    context.browser.find_by_xpath(submit_button_xpath_string).click()
+
+
+# Checking cookie values
+
+
+@step(
+    "the field {cookie_key} within the ons_cookie_policy cookie is set to {cookie_selection}"
+)
+def assert_ons_cookie_policy(context, cookie_key, cookie_selection):
+    ons_cookie_policy_dict = _parse_ons_cookie_policy_string(context)
+    actual_value = ons_cookie_policy_dict[cookie_key]
+    expected_value = True if cookie_selection == "On" else False
+
+    test_helper.assertEqual(actual_value, expected_value)
+
+
+@step("all optional cookies are set to {cookie_selection}")
+def assert_bulk_optional_cookie_selection(context, cookie_selection):
+    ons_cookie_policy_dict = _parse_ons_cookie_policy_string(context)
```

Comment:
I don't think you should pass in the entire context into the function, only what you need.  If you look at similar functions, we never really pass in the entire context object. Could you just pass in the cookies_dict into it?